Makefile

```diff
@@ -1,36 +1,32 @@
 .DEFAULT_GOAL := help
 
 APP_ID := talk_bot_ai
-JSON_INFO := "{\"id\":\"$(APP_ID)\",\"name\":\"Assistant Talk Bot\",\"daemon_config_name\":\"manual_install\",\"version\":\"1.0.0\",\"secret\":\"12345\",\"port\":10034,\"scopes\":[\"TALK\", \"TALK_BOT\"],\"system\":0}"
+APP_NAME := AssistantTalkBot
```

Comment:
`system` variable is an old thing, and this parameter is already ignored by AppAPI